packages/nodes-base/nodes/Taiga/Taiga.node.ts

```diff
@@ -0,0 +1,240 @@
+import {
+	IExecuteFunctions,
+} from 'n8n-core';
+
+import {
+	IDataObject,
+	ILoadOptionsFunctions,
+	INodeExecutionData,
+	INodePropertyOptions,
+	INodeType,
+	INodeTypeDescription,
+} from 'n8n-workflow';
+
+import {
+	taigaApiRequest,
+	taigaApiRequestAllItems,
+} from './GenericFunctions';
+
+import {
+	issueOperations,
+} from './IssueOperations';
+
+import {
+	issueOperationFields,
+} from './issueOperationFields';
+
+export class Taiga implements INodeType {
+	description: INodeTypeDescription = {
+		displayName: 'Taiga',
+		name: 'taiga',
+		icon: 'file:taiga.png',
+		group: ['transform'],
+		version: 1,
+		subtitle: '={{$parameter["operation"] + ": " + $parameter["resource"]}}',
+		description: 'Consume Taiga API',
+		defaults: {
+			name: 'Taiga',
+			color: '#772244',
+		},
+		inputs: ['main'],
+		outputs: ['main'],
+		credentials: [
+			{
+				name: 'taigaCloudApi',
+				displayOptions: {
+					show: {
+						version: [
+							'cloud',
+						],
+					},
+				},
+				required: true,
+			},
+			{
+				name: 'taigaServerApi',
+				displayOptions: {
+					show: {
+						version: [
+							'server',
+						],
+					},
+				},
+				required: true,
+			},
+		],
+		properties: [
+			{
+				displayName: 'Taiga Version',
+				name: 'version',
+				type: 'options',
+				options: [
+					{
+						name: 'Cloud',
+						value: 'cloud',
+					},
+					{
+						name: 'Server (Self Hosted)',
+						value: 'server',
+					},
+				],
+				default: 'cloud',
+			},
+			{
+				displayName: 'Resource',
+				name: 'resource',
+				type: 'options',
+				options: [
+					{
+						name: 'Issue',
+						value: 'issue',
+					},
+				],
+				default: 'issue',
+				description: 'Resource to consume.',
+			},
+			...issueOperations,
+			...issueOperationFields,
+		],
+	};
+
+	methods = {
+		loadOptions: {
+			// Get all the available tags to display them to user so that he can
+			// select them easily
+			async getTypes(this: ILoadOptionsFunctions): Promise<INodePropertyOptions[]> {
+				const project = this.getCurrentNodeParameter('project') as string;
+				const returnData: INodePropertyOptions[] = [];
+
+				const types = await taigaApiRequest.call(this, 'GET', `/issue-types?project=${project}`);
```

Comment:
Here I was filtering the issue types by the project id, so in order to use `projectSlug` we'll have to fetch the project id using it. To create the method `getProjectId` please see: https://taigaio.github.io/taiga-doc/dist/api.html#projects-get-by-slug
```suggestion
				const projectSlug = this.getCurrentNodeParameter('projectSlug') as string;
				const returnData: INodePropertyOptions[] = [];

				const project = await getProjectId(this, projectSlug);
				const types = await taigaApiRequest.call(this, 'GET', `/issue-types?project=${project}`);
```